### PR TITLE
Improve README a tad

### PR DESCRIPTION
**Pull request by thejcannon**
- Remove references to `flakehell`
- Restructure into written conventions & tooling
- Add a few "upcoming" sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,27 @@
...
-`ni-python-styleguide` serves several purposes:
-
-1. It is a `flake8` plugin that aggregates several other `flake8` plugins
-1. It houses our approved `flakehell` config: `flakehell_config.toml`
-1. When run directly, combines the above
+Welcome to NI's internal and external Python conventions and enforcement tooling.
 
 ## Written Conventions
 
 Our written conventions can be found at https://ni.github.io/python-styleguide/.
```
Comment (mshafer-NI) [inline README.md:13]: explicitly make this a link too??

**Review by mshafer-NI — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (thejcannon) [reply README.md:13]: To what, exactly?

Comment (mshafer-NI) [reply README.md:13]:
```suggestion
Our written conventions can be found at [ni.github.io/python-styleguide/](https://ni.github.io/python-styleguide/).
```

Comment (thejcannon) [reply README.md:13]: The Markdown renderer already linkifies it, and I like that someone could copy the text if they wanted.

I don't think it's worth changing, sorry.

**Review by neilvana — APPROVED** [review]

**Pull request merged**